lib/matplotlib/_api/deprecation.py

```diff
@@ -16,6 +16,42 @@
 import warnings
 
 
+# has to be here (instead of cbook) to prevent circular import, since
+# deprecation.deprecated is *used* during cbook's import
+class _classproperty:
```

Comment:
Since `_api` is already private.
```suggestion
class classproperty:
```

Also, this isn't actually a deprecation; does it belong in this file, or in `_api/__init__.py`?

Reply:
Most of the functions in `_api.deprecation` are still underscore delimited for direct importing into `cbook.deprecation`, happy to fix this one as long as you guys (@timhoffm) are cool with me sending in another PR to do this to the rest of the methods in `_api`, since I would feel gross if we end up with some random combination of underscores and no underscores.

Reply:
Oh, almost missed your other question. It should be in `__init__` now that `deprecation` isn't in `cbook` there's no more import-time circular stuff that can't be avoided by a local import.